utility/accessdata.py

```diff
@@ -12,9 +12,16 @@ def __init__(self, data_path_name):
         self.access_path = ''
         self._read_setup()
         self._set_private_connection()
-
-        if data_path_name is not None:
-            self.data_path_name = data_path_name
+        self._data_path_eval(path=data_path_name)
+
+    def _data_path_eval(self, path):
+        if path is not None:
```

Comment:
Maybe check if a None branch is needed.

Reply:
No. A None branch is not needed, but it would be good to have an add_path function if the data is inited with a None option.